src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (x) => {
+    return (y) => {
+      counter++;
+
+      if (counter > 3) {
+        counter = -2;
```

Comment:
do not need this
it is mylytsa
just use `%` and check if the counter is odd

Reply:
I have a question, if counter will go far away from 1, 2, 3, 4 rail - the counter in this app will eat more and more memory? 
I think in the way that 3, 4 -> -2 -> 2, 3, 4 -> -2 -> 2, 3, 4 will always hold only 3 primitive values in an app memory, isnt it better?

Reply:
In described situation - yes 👍 
Agree with you